django_6pm/django_6pm/django_6pm/settings.py

```diff
@@ -76,13 +77,13 @@
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'db_6pm',
-        'USER': 'postgres',
-        'PASSWORD': '',
-        'HOST': 'localhost',
-        'PORT': '5432',
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": "db_6pm",
+        "USER": "postgres",
+        "PASSWORD": "",
+        "HOST": "localhost",
+        "PORT": "5432",
```

Comment:
Why not using `.env` file?

Reply:
It is in the main django_6pm branch so thought that it will be added automatically once this branch will be merged to the main branch. Later, I will rebase product views branch to this master (django_6pm) one.